### PR TITLE
`mobile-tabs` - Fix layout on profiles

### PR DESCRIPTION
**Pull request by kovsu**
Close: #7910 

## Test URLs

https://github.com/refined-github/refined-github

https://github.com/refined-github

## Screenshot

image

image

https://github.com/user-attachments/assets/b42d6736-2672-4adc-b5fe-4e5a0dbcad2d

### CODE REVIEW

File: source/features/mobile-tabs.css [hunk 1]
```diff
@@ -67,7 +67,11 @@
 		justify-self: end;
 	}
 
-	:root .UnderlineNav-item .Counter {
+	:root .UnderlineNav-item span {
```
Comment (fregante) [inline source/features/mobile-tabs.css:70]:
```suggestion
	/* Tab label on on profiles */
	:root .UnderlineNav-item span:not([class]) {	
```

I'd also fix the extra padding and reduce the font size

**Comment by kovsu** [comment]
Updated:

CleanShot 2024-10-16 at 22 32 08@2x

CleanShot 2024-10-16 at 22 32 23@2x

**Comment by fregante** [comment]
It works on orgs but not on profiles 🥲 

https://github.com/fregante

Screenshot 10

**Comment by kovsu** [comment]
Everthing works fine now!

**Pull request merged**